scripts/config_setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+echo "Updating config files"
+
+echo "Moniker: $NODE_MONIKER"
+
+dasel put bool -f ${HOME}/.duality/config/app.toml -s ".api.enable" "true"
```

Comment:
I was going to ask where on earth did dasel come from in this Docker. But I realize now its from `ghcr.io/strangelove-ventures/infra-toolkit:v0.0.7` and that was very unclear. could we make a note either here or at import of dasel about what is happening?